aria/modeling/service_template.py

```diff
@@ -33,7 +33,8 @@
 from ..parser import validation
 from ..parser.consumption import ConsumptionContext
 from ..parser.reading import deepcopy_with_locators
-from ..utils import collections, formatting, console
+from ..utils import (collections, formatting, console)
```

Comment:
We have discussed this and have agreed that it can be in one line if it fits. We have a lot of code that is like this.